pkg/ring/kv/multi.go

```diff
@@ -40,8 +40,13 @@ type MultiRuntimeConfig struct {
 	// or to gossip). Doing this allows nice migration between stores. Empty values are ignored.
 	PrimaryStore string `yaml:"primary"`
 
-	// Mirroring enabled or not. Nil = no change.
-	Mirroring *bool `yaml:"mirror_enabled"`
+	// Mirroring enabled or not.
+	Mirroring bool `yaml:"mirror_enabled"`
```

Comment:
Would you be share the background on why changing the type is needed?

Reply:
I am interesting to know this too